### ISSUE TITLE
[HOLD for payment 2024-01-24] [$500] Thread - Edit box with parent message not in focus when there many comments in thread

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#30805]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:**1.3.95-0
**Reproducible in staging?:**Y
**Reproducible in production?:**Y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:**Applause - Internal Team
**Slack conversation:**

## Action Performed:
1. Open the app
2. Navigate to any chat
3. Send a message and start a thread
4. Send some messages > 14
5. Long tap the parent message
6. Select "Edit comment"

## Expected Result:
The edit box with the parent message is in focus, and the user is able to see it

## Actual Result:
The edit box with the parent is not in focus until the user starts typing

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

</details> 

https://github.com/Expensify/App/assets/78819774/cbee253a-dd51-4848-bf47-0af4f66acb31

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01c7c8ad57a6053cea</li>
        <li>Upwork Job ID: 1720205591610355712</li>
        <li>Last Price Increase: 2023-11-02</li>
<li>Automatic offers: </li>
<ul>
<li>aimane-chnaif | Reviewer | 28062420</li>
<li>Krishna2323 | Contributor | 28062421</li>
</ul></li>
    </ul>
</details>

**Comment by abekkala** [comment]
still holding on this one

**Comment by abekkala** [comment]
still holding on 30729

**Comment by abekkala** [comment]
still on hold

**Comment by abekkala** [comment]
Still waiting on 30729
I'll be ooo until Mon 11/27 and will check on this one at that time.

**Comment by aldo-expensify** [comment]
Still waiting on 30729

**Comment by aldo-expensify** [comment]
Still waiting on [30729](https://github.com/Expensify/App/issues/30729)

**Comment by abekkala** [comment]
Fix for 30729 merged to production 4 days ago (2013-12-07)

**Comment by abekkala** [comment]
We're good to continue here. @aldo-expensify do you agree with @aimane-chnaif that the proposal by @Krishna2323 will resolve this?

**Comment by abekkala** [comment]
@aimane-chnaif can you please complete the checklist above?

**Comment by abekkala** [comment]
@Krishna2323 payment sent and contract ended - thank you! 🎉

**Comment by abekkala** [comment]
@aimane-chnaif payment sent and contract ended - thank you 🎉
